e2e/cli-e2e/tests/__snapshots__/help.e2e.test.ts.snap

```diff
@@ -16,9 +16,11 @@ Options:
                                                        [boolean] [default: true]
       --verbose              When true creates more verbose output. This is help
                              ful when debugging.      [boolean] [default: false]
-      --config               Path the the config file, e.g. code-pushup.config.t
-                             s. By default it loads code-pushup.config.(ts|mjs|j
-                             s).                                        [string]
+      --config               Path to config file, e.g. code-pushup.config.ts. By
+                              default it loads code-pushup.config.(ts|mjs|js).
+                                                                        [string]
+      --tsconfig             Path to a TypeScript config, to be used when loadin
```

Comment:
Can you add this option to `print-config` E2E tests or a `yargs-cli` integration test to check that passing this argument is correctly registered by the CLI?